charts/netbox/README.md

```diff
@@ -155,6 +155,7 @@ The following table lists the configurable parameters for this chart and their d
 | `remoteAuth.ldap.serverUri`                     | see [django-auth-ldap](https://django-auth-ldap.readthedocs.io)     | `""`                                         |
 | `remoteAuth.ldap.startTls`                      | if StarTLS should be used                                           | *see values.yaml*                            |
 | `remoteAuth.ldap.ignoreCertErrors`              | if Certificate errors should be ignored                             | *see values.yaml*                            |
+| `remoteAuth.ldap.caCertDir`                     | CA certificate directory                                            | *see auth.md*                                |
```

Comment:
This must be added to the values (`values.yaml`) and the schema (`values.schema.json`).